toolkit/pythonit_toolkit/service_client/client.py

```diff
@@ -0,0 +1,61 @@
+from dataclasses import dataclass
+from typing import Any, Dict, Optional, Union
+
+import httpx
+from pythonit_toolkit.headers import SERVICE_JWT_HEADER
+from pythonit_toolkit.pastaporto.tokens import generate_service_to_service_token
+
+
+@dataclass
+class ServiceResponse:
+    data: Optional[Dict[str, Any]]
+
+
+@dataclass
+class ServiceError(Exception):
+    errors: Dict[str, Any]
+
+
+class ServiceClient:
+    def __init__(
+        self,
+        url: str,
+        issuer: str,
+        audience: str,
+        jwt_secret: str,
```

Comment:
can you do the changes I suggested here: https://github.com/pythonitalia/pycon/pull/2431#discussion_r748662345?